doc/index.rst

```diff
@@ -11,41 +11,47 @@ DeePMD-kit is a package written in Python/C++, designed to minimize the effort r
 
 .. Important:: The project DeePMD-kit is licensed under `GNU LGPLv3.0 <https://github.com/deepmodeling/deepmd-kit/blob/master/LICENSE>`_. If you use this code in any future publications, please cite this using *Han Wang, Linfeng Zhang, Jiequn Han, and Weinan E. "DeePMD-kit: A deep learning package for many-body potential energy representation and molecular dynamics." Computer Physics Communications 228 (2018): 178-184.*
 
-.. _user-guide:
+.. _getting-started:
 
 .. toctree::
-   :maxdepth: 2
-   :caption: User Guide
+   :maxdepth: 3
+   :caption: Getting Started
    
-   install
-   getting-started
-   tensorboard
-   troubleshooting/index
-
+   getting-started/index
 
-.. _data-and-parameters:
+.. _advanced:
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Data and Parameters
-
-   data-conv
-   train-input
-
+   :maxdepth: 3
+   :numbered:
+   :caption: Advanced
+
+   install/index
+   data/index
+   model/index
+   train/index
+   freeze/index
+   test/index
+   inference/index
+   third-party/index
+   troubleshooting/index
 
 .. _developer-guide:
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 3
    :caption: Developer Guide
    
-   development/index
+   api
```

Comment:
And you can change here to
```python
.. toctree
    :maxdepth: 3
    :caption: Developer Guide
    :glob:

    development/*
```

Reply:
But, sections will be unordered?

Reply:
Yes, but I think "trouble shooting" is a problem pool for people to mine in, so sections may not be necessary?

Reply:
The sections won't change much. Originally, the list was like this 
```python
.. toctree::
   :maxdepth: 3
   :caption: Developer Guide

   api
   API_CC/api_cc
   development/coding-conventions
   development/type-embedding
```
The automatically generated one with ":glob:" is exactly the same.